### ISSUE TITLE
Default configuration runs with system Electron rather than local Electron

### ISSUE THREAD

**Issue by rmunn** [fable-compiler/samples-electron#29]
The `launch` configuration in `package.json` currently contains `electron .`, which will run the first version of Electron found on the system PATH. This could potentially be an older version of Electron, or it might even fail if someone doesn't have Electron installed globally. For example, on my system, I had Electron 1.4.10 installed globally, which uses an older version of Chromium that doesn't include CSS Grid. If I had wanted to design my UI layout with Grid (which makes many UI layouts SO MUCH easier), I would have had problems.

Switching the `launch` configutation to be `node_modules/electron/cli.js .` instead of `electron .` should solve this problem (at least it works for me on Linux, though perhaps that should be tested on Windows and Mac as well), and let anyone run the locally-built version of Electron without needing to have a global Electron installation.

**Comment by aolney** [comment]
On my system (Debian) `electron .` works fine with local electron (my understanding is that best practice is to **not** install globally).

What you propose sounds reasonable, but it's not clear to me that this is a bug rather than a point of preference. 

For example, if I look at the [electron quick start](https://github.com/electron/electron-quick-start/blob/master/package.json), they use `electron .` 

This makes me think the current approach is the de facto standard.

**Comment by MangelMaxime** [comment]
I believe when using cli commands from packages.json, it's should resolve the locally packages first and then go to the global installation if not found.

Did you follow the instructions ? https://github.com/fable-compiler/samples-electron#building-and-running-the-app

**Comment by alfonsogarciacaro** [comment]
Yes, my understanding is what @MangelMaxime says, the cli commands in package.json script will be resolved locally first and then globally so I guess it's fine. Also, as @aolney says, the package.json was actually copied from Electron quickstart.

I'll close the issue, but please feel free to reopen it @rmunn if you still have other questions.

**Comment by rmunn** [comment]
That was on my desktop at work, and I'm currently at home. On my Linux box at home, `electron .` in the package.json does indeed run the locally-installed electron, not the globally-installed one. I must have an environment variable set on my desktop at work that is causing a different search path for `yarn run`, but I won't have access to that machine again until tomorrow. I'll leave this bug closed, and investigate tomorrow to see if I can figure out which environment variable I set that causes the different behavior — it might be worth mentioning in the README. (At the time I set that environment variable, whichever one it was, I wasn't yet very familiar with npm and yarn, and I was reading instructions off the Web to try to get `npm install -g` to install into my home folder so I didn't have to sudo. One of those may have told me to set a Node search path variable, and I don't know which one).

Anyway, as I said, I'll leave this bug closed until I know exactly how I caused the problem to occur, then I'll either reopen this issue if it really is a bug, or else submit a PR for a note in the README if it's just a case of "Hey, if you have this non-standard configuration, you might need to unset environment variable XYZ."
